web/src/components/App.scss

```diff
@@ -0,0 +1,58 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,600;0,700;0,800;1,300;1,400;1,600;1,700;1,800&display=swap');
+
+$base-font: 'Open Sans', sans-serif;
+
+$bg-color: rgb(245, 246, 250);
+$eduace-gray: rgb(160, 160, 160);
+
+html {
+  background-color: $bg-color;
+  font-family: $base-font;
+}
+
+body {
+  margin: 0
+}
+
+.eduace-app-container {
+  display: grid;
+  grid-template-columns: repeat(10, 10%);
+  min-height: 100vh;
+}
+
+.eduace-content-container {
+  grid-column: 3 / 10;
```

Comment:
I haven't seen this grid stuff before but if it works how I think it does, that's super cool

Reply:
HTML5 Grid is the future (until the next big thing) aye